baseapp_mentions/services.py

```
@@ -1,95 +1,282 @@
+    ) -> List["Profile"]:
+        """Resolve Relay global IDs to Profile instances, filtering out self-mention.
+
+        Silently drops malformed or stale IDs so a flaky client reference does not
+        break the parent mutation.
+        """
+        pks: List[int] = []
+        for relay_id in mentioned_profile_ids or []:
+            try:
+                pk = get_pk_from_relay_id(relay_id)
+            except Exception:  # noqa: BLE001 — malformed IDs are dropped intentionally
+                continue
```

Ruff (0.15.13)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)